app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -205,13 +205,12 @@
     <string name="message_user_joined_channel">Entrou na sala</string>
     <string name="message_welcome">Bem-vindo, %s</string>
     <string name="message_removed">Mensagem removida</string>
-    <string name="message_pinned">Pinou uma mensagem:</string>
+    <string name="message_pinned">Fixou uma mensagem:</string>
     <string name="message_muted">Usuário %1$s foi silenciado por %2$s</string>
     <string name="message_unmuted">Usuário %1$s saiu do modo silenciado por %2$s</string>
-    <string name="message_role_add">%1$s foi definido %2$s por %3$s</string>
-    <string name="message_role_removed">%1$s não é mais %2$s por %3$s</string>
-    // TODO:Add proper translation.
-    <string name="message_credentials_saved_successfully">Credentials saved successfully</string>
+    <string name="message_role_add">%3$s colocou o estatuto de %2$s a %1$s</string>
```

Comment:
Why did you invert the order of the params here?

Reply:
I think it makes more sense something like:
@TheMythPT colocou o estatuto de administrador a @filipedelimabrito 
Probably changing `colocou` by `deu`?

Reply:
When I get home, I'll make a few reviews.